### PR TITLE
HTML TASK

### PR DESCRIPTION
**Pull request by Heedharyhart**
I closed all my formal PR and resubmitting my HTML task
I added my registration page with CSS styling

### CODE REVIEW

File: submissions/Idayatolaide/CSS TASK1/HTML.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```
Comment (kenny-wealth) [inline submissions/Idayatolaide/CSS TASK1/HTML.html]: Nice work. Please remove the space in folder name

File: submissions/Idayatolaide/CSS TASK1/HTML.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>WeddingBellZ website</title>
+    <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+    <h1>WeddingBellZ</h1>
+    <p>WeddingBellZ is an online Marketplace where couples about to get Married find a designers for there <br>Traditional Asooke wears
```
Comment (kenny-wealth) [inline submissions/Idayatolaide/CSS TASK1/HTML.html]: Indentation seems off

File: submissions/Abdulkareem Idayat Olaide [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (kenny-wealth) [inline submissions/Abdulkareem Idayat Olaide:1]: This file is empty, you may delete it if it is not needed

File: submissions/Idayatolaide/CSS TASK1/HTML.html (hunk 1 above)
Comment (Heedharyhart) [reply submissions/Idayatolaide/CSS TASK1/HTML.html]: I have removed the spacing

Comment (Heedharyhart) [reply submissions/Idayatolaide/CSS TASK1/HTML.html]: I newly added my landing page

Comment (giselle2323) [reply submissions/Idayatolaide/CSS TASK1/HTML.html]: Well done.  Please add a style to the list tags, they are not showing when viewed in the browser.

**Pull request closed without merging**